resources/views/components/quill-js.blade.php

```diff
@@ -0,0 +1,35 @@
+@if (config('firefly.features.wysiwyg'))
+    <script>
+        var toolbarOptions = [
+            ['bold', 'italic', 'underline', 'strike'],        // toggled buttons
+            ['blockquote', 'code-block'],
+
+            [{ 'header': 1 }, { 'header': 2 }],               // custom button values
+            [{ 'list': 'ordered'}, { 'list': 'bullet' }],
+            [{ 'script': 'sub'}, { 'script': 'super' }],      // superscript/subscript
+            [{ 'indent': '-1'}, { 'indent': '+1' }],          // outdent/indent
+            [{ 'direction': 'rtl' }],                         // text direction
+
+            [{ 'size': ['small', false, 'large', 'huge'] }],  // custom dropdown
+            [{ 'header': [1, 2, 3, 4, 5, 6, false] }],
+
+            [{ 'color': [] }, { 'background': [] }],          // dropdown with defaults from theme
+            [{ 'font': [] }],
+            [{ 'align': [] }],
+
+            ['clean']                                         // remove formatting button
+        ];
+        var options = {
+            modules: {
+                toolbar: toolbarOptions
+            },
+            theme: 'snow'
+        };
+        var editor = new Quill('#content',options);
+
+        document.getElementById('submit').addEventListener('click',function(event) {
+            var content = document.getElementsByClassName('ql-editor')[0].innerHTML
+            document.getElementById('content_hidden').setAttribute('value', content)
+        })
```

Comment:
What is this for exactly? 🤔

Reply:
Weirdly enough, Quill provides the WYSIWYG content editor, but uses a div, not a textarea, so does not hook into the form submission directly. This takes the content from Quill, and puts it in a hidden form field on submit to send it back o the server with the form submission.